README.md

```diff
@@ -67,9 +67,11 @@ Refer to [**aantron/bisect-starter-dune**][dune-repo], which produces
     (* -*- tuareg -*- *)
 
     let preprocess =
-      match Sys.getenv "BISECT_ENABLE" with
-      | "yes" -> "(preprocess (pps bisect_ppx))"
-      | _ | exception _ -> ""
+      try
+        match Sys.getenv "BISECT_ENABLE" with
+        | "yes" -> "(preprocess (pps bisect_ppx))"
+        | _ -> ""
+      with _ -> ""
```

Comment:
Would a separate `| exception Invalid_argument _ ->` case (not an or-pattern) still work, and be more tidy?

Reply:
I'm not sure what you're suggesting. The issue that this PR is fixing is that `| exception e ->` is only valid since OCaml 4.07. This pattern did not exist before.

Otherwise feel free to change my code if there is a coding style that you prefer.

Reply:
`| exception e -> ...` was added in [4.02](https://caml.inria.fr/pub/docs/manual-ocaml/patterns.html#sss:exception-match). What was not allowed (it seems) is using it inside an or-pattern, as opposed to at the top level of a case. So `match foo with | exception e -> ...` was ok, but `match foo with | p | exception e -> ...` was not.

Reply:
Ohhh I thought it was added much later. Sorry ^^"

Reply:
Meanwhile, I had always assumed it was only allowed at the top level since 4.02, tried it in an or-pattern on a whim for the README, and failed to realize that Bisect_ppx itself doesnt handle this extension :p